### ISSUE TITLE
Question regarding use of commented-out variables

### ISSUE THREAD

**Issue by bgro** [ansible-lockdown/UBUNTU22-CIS#82]
**Question**
`defaults/main.yml` contains some variables that are commented out and must be uncommented 
in order to be used. For example:

~~~
# ubtu22cis_journald_systemmaxuse: SystemMaxUse=
# ubtu22cis_journald_systemkeepfree: SystemKeepFree=
# ubtu22cis_journald_runtimemaxuse: RuntimeMaxUse=
# ubtu22cis_journald_runtimekeepfree: RuntimeKeepFree
# ubtu22cis_journald_maxfilesec: MaxFileSec=
~~~

and 

~~~
ubtu22cis_sshd:
    (...)
    # WARNING: make sure you understand the precedence when working with these values!!
    allow_users: "vagrant ubuntu"
    allow_groups: "vagrant ubuntu"
    # deny_users:
    # deny_groups:
~~~

Seeeing variables that are commented out always raises the question, whether these really can be used
or are maybe remants of ongoing development. Also, they make it harder to add additional information
in comments, since then what is potentially functional code and what is documentation becomes blended.

I understand the usage of this approach for the variables concerning `journald`, because otherwise the
current implementation of placing a commented-out line in the file as default value does not work any more.

(Though I think this could be solved by using two stages -- one for writing the respective lines and a second
one for "mopping up" the lines with empty values, replacing, e.g. `^SystemMaxUse=\s*$` with `#SystemMaxUse=` --
that would also remove the need to supply the *full* line rather than just the desired value, the only occurrence of such
an approach in this role, I think.)

But for `deny_users`and `deny_groups`, if I don't misunderstand the implementation, also empty default values
should work:

~~~
ubtu22cis_sshd:
    (...)
    # WARNING: make sure you understand the precedence when working with these values!!
    allow_users: "vagrant ubuntu"
    allow_groups: "vagrant ubuntu"
    deny_users:
    deny_groups:
~~~

Should I make a change/pull request regarding the uncommenting of these two variables, or am I overlooking something?

**Environment (please complete the following information):**

- Ansible Version: [e.g. 2.10]
- Host Python Version: [e.g. Python 3.7.6]
- Ansible Server Python Version: [e.g. Python 3.7.6]
- Additional Details:

**Comment by uk-bolly** [comment]
hi @bgro 

Great issue and questions, have been looking and testing this today. I have worked through the journald question and set defaults in the defaults/main.yml with a comment. 
For the sshd i have tested and we are unable to add empty values to the config itself. I have rewritten this to a more standard approach and again removed the commented variables to useable.
Thank you for highlighting this and how this is interpreted by users, this really does help us to improve the content so more available and useable by all.
I have added the changes to the sept_issues branch.

many thanks again

uk-bolly
